### DISCUSSION TITLE
jpg or png import into svg

### DISCUSSION THREAD

**Discussion by shieldsit** [ExperienceLovelace/ha-floorplan#217] [General]
Is it possible to do this?  I've been trying to get a snapshot from my cameras visible in my floor plan but so far I'm not having any luck.  I've seen an example of doing this in the examples, but I think it was importing an svg into a svg.

**Comment by exetico** [comment]
No, you cant convert it to frames and objects. But you can for sure add a static image, if you're trying to get a mock-up running (and add image as a background). 

Check this post:
https://github.com/ExperienceLovelace/ha-floorplan/discussions/131#discussioncomment-1654167

Sone hard work is required 👍😊.

**Reply by shieldsit** [reply]
Thanks very much for the info!  I think I will just forget about it for now!  :)
